### PR TITLE
Improve minio webhook

### PR DESCRIPTION
**Pull request by aliok**
- Use a slim base image
- Use gunicorn production server
- Update instructions in README
- Add graceful shutdown

### CODE REVIEW

File: infra/openshift-manifests/minio-webhook-source/broker.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ kind: Broker
 metadata:
   name: predictions-broker
   namespace: ai-demo
+  annotations:
+    eventing.knative.dev/broker.class: MTChannelBasedBroker
```
Comment (pierDipi) [inline infra/openshift-manifests/minio-webhook-source/broker.yaml:7]: Why this, just use whatever default is configured IMHO

File: infra/openshift-manifests/minio-webhook-source/ksvc.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ spec:
       containers:
         - image: quay.io/kevent-mesh/ai-demo-minio-webhook-source:main
           ports:
-            - containerPort: 5000
+            - containerPort: 8080
```
Comment (pierDipi) [inline infra/openshift-manifests/minio-webhook-source/ksvc.yaml:14]: 8080 is the default, so we can drop it

Comment (aliok) [reply infra/openshift-manifests/minio-webhook-source/ksvc.yaml:14]: hm, I create a new task for this, to drop it in all services

File: infra/openshift-manifests/minio-storage-configuration.yaml [hunk 3]
```diff
@@ -11,4 +11,4 @@ stringData:
     export MINIO_STORAGE_CLASS_STANDARD="EC:0"
     export MINIO_BROWSER="on"
     export MINIO_NOTIFY_WEBHOOK_ENABLE_PRIMARY="on"
-    export MINIO_NOTIFY_WEBHOOK_ENDPOINT_PRIMARY="http://event-display.ai-demo.svc.cluster.local:80"
+    export MINIO_NOTIFY_WEBHOOK_ENDPOINT_PRIMARY="http://minio-webhook-source.ai-demo.svc.cluster.local"
```
[the remark this answers is not in the document]
Comment (aliok) [reply infra/openshift-manifests/minio-storage-configuration.yaml]: reverted this change

**Pull request merged**